include/seqan3/core/detail/strong_type.hpp

```diff
@@ -64,12 +65,65 @@ constexpr bool add_enum_bitwise_operators<seqan3::detail::strong_type_skill> = t
 namespace seqan3::detail
 {
 
+/*!\interface seqan3::detail::strong_type_specialisation <>
+ * \brief Defines the requirements of a seqan::detail::strong_type specialisation.
+ * \tparam strong_type_t The type the concept check is performed on (the putative strong type).
+ * \ingroup core
+ */
+/*!\name Requirements for seqan3::detail::strong_type_specialisation
+ * \brief You can expect these members on all types that implement seqan3::detail::strong_type_specialisation.
+ * \relates seqan3::detail::strong_type_specialisation
+ * \{
+ */
+//!\cond
+template <typename strong_type_t>
+SEQAN3_CONCEPT strong_type_specialisation = requires (strong_type_t && obj)
+{
+//!\endcond
+
+    /*!\typedef typedef IMPLEMENTATION_DEFINED value_type;
+     * \brief The underlying type represented by this strong type.
+     */
+    typename std::remove_reference_t<strong_type_t>::value_type;
+
+    /*!\var static constexpr seqan3::detail::strong_type_skill skills;
+     * \brief The selected skills for this strong type.
+     */
+    { std::remove_reference_t<strong_type_t>::skills };
+    //!\cond
+    requires std::same_as<decltype(std::remove_reference_t<strong_type_t>::skills), strong_type_skill const>;
```

Comment:
:crying_cat_face: This will mess up #1866 again :crying_cat_face: But such is live :shrug: